explorer/site-config/hca-dcp/dev/projectsEntity.ts

```diff
@@ -38,6 +38,27 @@ export const projectEntity = {
           transformer: T.projectsToSpeciesColumn,
```

Comment:
Can you refactor the top-level `export const projectEntity` to be `projectsEntity`? This will match the naming convention of files and samples, as well as the name of this file.